src/Layouts/stars/feed/note.tsx

```diff
@@ -0,0 +1,108 @@
+import { ReplyOutlined, DeleteOutline } from "@material-ui/icons";
+import { Grid, Avatar, Typography, Divider } from "@mui/material";
+import { useState } from "react";
+import { noteType } from "../../../assets/star";
+
+interface noteProps {
+  notes: noteType[];
+  note: noteType;
+  replies: noteType[];
+  replyBranch: number;
+  replyTo: noteType | undefined;
+  setReplyTo: (param: noteType | undefined) => void;
+  deleteNote: (note: noteType) => void;
+}
+const Note = ({
+  notes,
+  note,
+  replies,
+  replyBranch,
+  replyTo,
+  setReplyTo,
+  deleteNote,
+}: noteProps) => {
+  const [isReply, setIsReply] = useState(false);
+
+  const indent = (replyBranch + 40).toString() + "px";
+
```

Comment:
not important  at all but 
```
const getReplies = (note: noteType) =>notes.filter((n: noteType) => n.repliesTo === note.id)
```
look why better then 
```
 const getReplies = (note: noteType) => {
    return notes.filter((n: noteType) => n.repliesTo === note.id);
  };
```

Reply:
all line length stuff have been fixed with linter